content/en/metrics/guide/dynamic_quotas.md

```diff
@@ -0,0 +1,86 @@
+
+## Setup
+
+### Prerequisites
+
+1. A Datadog account with [Workflow Automation][3] enabled
+2. The Datadog [Slack integration][5] installed
+
+### Configure blueprint
+
+#### How to build a per-metric quota
+
+{{< img src="/metrics/guide/dynamic_quotas/automated_mwl_workflow_monitor.png" alt="The Make a decision tiles on the Datadog automated Metrics without Limits™ workflow blueprint" style="width:100%;" >}}
+
+1. Open this [workflow blueprint][8], and click "Create from Blueprint"
+2. Click on the green Monitor tile, and enable the "Automatic triggering" toggle. Also copy the **Mention handle** for this workflow -- you'll use this in Step 5 below
+3. Create a [metric monitor][9]. In the **Define the metric** step, select ```datadog.estimated_usage.metrics.custom.by_metric``` as your metric name and choose the ```SUM BY``` space aggregator (recommendd monitor type - Threshold Alert, but you can also use Change or Anomaly Detection)
+4. Under ***Set alert conditions***, define your quota threshold number
+5. Under ***Configure notifications & automations***, you'll update the monitor notification message and include the Workflow handle from Step 2) above
+6. Select "Multi Alert" to send a notification for each culprit metric.
+7. Click ***Create*** to create the metric monitor
+8. Head back into the Workflow blueprint and click on the tiles titled ***Make a decision*** and ***Make a decision 1***. 
+
+{{< img src="/metrics/guide/dynamic_quotas/slack_decisions_1.png" alt="The Make a decision tiles on the Datadog automated Metrics without Limits™ workflow blueprint" style="width:100%;" >}}
+
+You'll be asked to choose the Slack workspace and either a Slack channel or specific user who will be notified when the monitor alerts. These are the users who will be asked to give Datadog permission to automatically reduce your metrics costs on your behalf
+
+9. For the ***Send message*** and ***Send message 1*** tiles, you'll also need to select the Slack workspace. You'll also be asked whether you want to notify a channel or specific user after Datadog has made the cost saving configurations on your behalf
+
+{{< img src="/metrics/guide/dynamic_quotas/slack_decisions_2.png" alt="The Send message tiles on the Datadog automated Metrics without Limits™ workflow blueprint" style="width:100%;" >}}
+
+10. Click ***Save***.
+11. Click ***Publish*** and ***Run*** to start automatically managing your custom metrics costs. 
+
+
+#### Configure Slack notifications
```

Comment:
Is there a reason we still need this section? you walk folks through the decision points where they need to add the channel or user(s) that grant permission

Reply:
Agreed, steps 8 - 9 and their images aren't necessary -- will remove.